pydust/buildzig.py

```diff
@@ -38,11 +37,13 @@ def zig_build(argv: list[str], conf: config.ToolPydust | None = None):
     conf = conf or config.load()
 
     # Always generate the supporting pydist.build.zig
-    update_file(conf.pydust_build_zig, generate_pydust_build_zig())
+    with open(conf.pydust_build_zig, "w+") as f:
```

Comment:
I think since we don't actually use the contextmanager for anything you can just use https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text and write_text

Reply:
We can't, since we need `w+` not `w`

Reply:
Where do we read the file?